const tryCatch = require('try-catch');

test('some test', (t) => {
    t.throws(copymitter, /from should be a string!/, 'should throw when no args');
    t.end();
});
